frontend/src/styles/css-variables.ts

```
@@ -0,0 +1,341 @@
+  return `/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/**/*.{js,ts,jsx,tsx,mdx}',
+  ],
+  theme: {
+    extend: {
+      colors: ${JSON.stringify(designSystem.colors, null, 2)},
+      fontFamily: ${JSON.stringify(designSystem.typography.fontFamily, null, 2)},
+      fontSize: ${JSON.stringify(
+        Object.entries(designSystem.typography.fontSize).reduce((acc, [key, value]) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)